### DISCUSSION TITLE
[Feature request]: support gestures using more fingers

### DISCUSSION THREAD

**Discussion by belatent** [ClassicOldSong/moonlight-android#31] [Ideas]
### Is your feature request related to a problem? Please describe.

First of all, a huge thanks for adding two finger support for the touchpad! It completely fixed this senario on my oneplus pad 2, and it works surprisingly well. Finally found an alternative that's willing to handle the touchpad!

However, gestures using more fingers are still not working. Features like going back to the desktop and quickly switching windows all rely on that.

### Describe the solution you'd like

Actually, my device supports up to five finger gestures. And it looks like your last update made a general fix that happened to cover mine as well. So I guess we're just a couple of tweaks away from perfecting this. Any luck to make full windows gesture support happen? 

### Describe alternatives you've considered

-

**Comment by ClassicOldSong** [comment]
If you're using local cursor mode, then sadly there's no way to add more gestures. It's Android's janky event system that making things very hard to work with. More gestures are feasible theoratically when not using local cursor mode, but that requires a complete rewrite from original Moonlight's multi-touch handler, which could take some time.

The handling is already a mess with the current implementation for scrolling and dragging....

For returning to desktop, you can try Win+D, and for switching windows, try Win+Tab, as Alt+Tab might be captured by Android system.
